sources/packages/backend/libs/sims-db/src/entities/notification-metadata.type.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Metadata for notification.
+ */
+export interface Metadata {
+  disbursement_id: number;
```

Comment:
As metadata, I would suggest having it defined as `disbursementId`. 
In the context where it is it should be ok to call it only `Metadata` but I am afraid that it will be too generic.
Can we rename it as `NotificationMetadata`?